packages/frontend-v2/package.json

```diff
@@ -9,7 +9,11 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@chakra-ui/react": "^1.8.8",
```

Comment:
another thing to think about is not all of our components will be chakra components–how do we apply styles to the non-chakra components? do we use emotion?

if so, then now there will be multiple methods of writing styles in our codebase (styled-components-like way using emotion, and via chakra UI props) that devs need to be familiar with before being able to understand code.

Reply:
hmmm... so having used chakra quite a bit, I actually have never needed to write a custom component? I could totaaalllyyy be missing cases where we would need to so let me know!

The whole point of Chakra is to have a single source of truth for styling, even the most basic components are available and can be styled using chakra(divs, paragraphs, etc) almost as extensively as one could style with css. 

The single source of truth makes it much easier to have a unified theme and responsive behaviour that we define in one file and all components behave accordingly. For instance, instead of using a HTML <button> we use <Button> from chakra everywhere, that way if our need to change how a button behaves responsively/sizing(update what "small" means etc), we can do so in the theme file and have it update everywhere. This is different from have a styled.button and using theme values since with Chakra everything is implicitly themed.

Essentially, using Chakra components for even the simplest styles by nature gives us control to every single component on our pages from a single file without having to explicitly use theme values.

Reply:
yeah, makes sense. typically when implementing designs from figma though, small things end up being slightly off (ie, the spacing in design doesn't _quite_ match the spacing defined in theme–though this example is easy to take care of using chakra).

though, it is good to hear that you haven't had to dip into other ways of styling when using chakra